modules/littlefs/zephyr_lfs_crc.c

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
The file is in the zephyr littlefs module, so the zephyr_lfs_ prefix is just noise IMHO.

Should rename to crc.c